crafters/image/ImageResizer/tests/test_imageresizer.py

```diff
@@ -10,7 +10,15 @@ def test_resize():
     img_width = 20
     img_height = 17
     output_dim = 71
+    
+    # Test for int target_size
     crafter = ImageResizer(target_size=output_dim)
-    img_array = create_random_img_array(img_width, img_height)
+    img_array = create_random_img_array(img_height, img_width)
```

Comment:
I think the point is to change the order in the function arguments and not in the call.

Reply:
no my bad!

Reply:
Couldn't get you. The target_size=int will resize any of the smaller size. I changed this to match the conventions of the `random array` function and tuple of target_dim which expects `(height, width)`

Reply:
yes yes, my bad, I always get my head messy with num_rows, num_cols, widths and heights!